### PR TITLE
feat(apps): Update python commands

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

Fixes https://github.com/meroxa/turbine-project/issues/83

## Type of change

- [x]  New feature
- [ ]  Bug fix
- [x]  Refactor
- [ ]  Documentation

## How was this tested?

- [ ]  Unit Tests
- [x]  Tested in staging
- [ ]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|||

### CODE REVIEW

**Review by ericcheatham — APPROVED** [review]
🎉  🌮

**Comment by janelletavares** [comment]
going to merge this after the related turbine-py PR is merged

**Comment by raulb** [comment]
Love this starting to come together.

**Review by raulb — APPROVED** [review]

**Pull request merged**
